### PR TITLE
[WIP] Personal Information Survey.

### PR DESCRIPTION
**Pull request by duckinator**
TODO:
- [ ] Actually write the survey.
- [ ] Get feedback from other people.

### CODE REVIEW

**Comment by duckinator** [comment]
At this point, the survey is documented, but doesn't actually exist.

**Comment by duckinator** [comment]
You can read it here: https://github.com/duckinator/employee-handbook/blob/employee-information-survey/src/03-personal-information-survey.md

File: src/03-personal-information-survey.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Personal Information Survey
+
+The Personal Information Survey is a brief questionnaire that is answered before the interview process starts, and can be updated as necessary. The purpose of the Survey is to help employees and interviewees more respectfully communicate with each other. It includes such information as:
+
+1. **Name** to use when referring to and addressing them.
+2. **Pronouns** to use when referring to them.
+3. **Title** to use when addressing them in a formal context.
```
Comment (zspencer) [inline src/03-personal-information-survey.md]: What about form of address? Mx or M or ?

File: src/03-personal-information-survey.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+4. **Time zone** they live in, so we can more effectively plan meetings.
```
Comment (zspencer) [inline src/03-personal-information-survey.md]: We can use ScheduleOnce for this.

**Comment by zspencer** [comment]
:+1:

Where should the results live? Google Drive? Maybe an Employee Directory summarizing the response with a link to each persons individual page?

File: src/03-personal-information-survey.md (hunk 1 above)
Comment (duckinator) [inline src/03-personal-information-survey.md]: That's what I meant by title. Is title not the right word for that?

Comment (zspencer) [inline src/03-personal-information-survey.md]: Oh, I guess? Maybe?

**Comment by duckinator** [comment]
Long term, I'm not sure. The easiest one to configure would probably be a wiki of some sort. If we make a GitHub repo accessible to all employees and use the accompanying wiki, then we can just have a table like:

```
Name | Pronouns | Title
-----|----------|------
Marie Markwell | she/her/herself or they/them/themself | Ms or Mx
```

and it would expand to this:

Name | Pronouns | Title
-----|----------|------
Marie Markwell | she/her/herself or they/them/themself | Ms or Mx

Then, adding new employees to it is as simple as appending:

```
<name> | <pronouns> | <title>
```

File: src/03-personal-information-survey.md (hunk 1 above)
Comment (duckinator) [inline src/03-personal-information-survey.md]: I [looked it up](http://en.wikipedia.org/wiki/Title#Titles_in_English-speaking_areas) and it is.

File: src/03-personal-information-survey.md (hunk 2 above)
Comment (duckinator) [inline src/03-personal-information-survey.md]: ScheduleOnce or similar seems far more reasonable, especially considering the nonsense that is timezones and daylight savings. :thumbsup:

**Comment by zspencer** [comment]
Easiest to configure would be a Google Drive spreadsheet; and it would be more accessible to non-programmer team members. I've created it here: https://docs.google.com/a/zincma.de/spreadsheets/d/1MGZJidkiasWWA98JugD6kmvRG1i66XOavelLs050D_k/edit#gid=0

(This is only accessible to members of Zinc)

**Comment by zspencer** [comment]
<3 Appreciate you @duckinator !

**Pull request closed without merging**